src/shared/components/carousel/carousel-videos-item/index.js

```diff
@@ -5,36 +5,36 @@ import PropTypes from 'prop-types'
 
 import styles from './index.module.css'
 
-class RemainingVideo extends Component {
+class CarouselVideosItem extends Component {
```

Comment:
I would prefer having a generic CarouselItem, which received a children that would be the `ReactPlayer` in this case. Just add it to your list, and do it if you have time in the end.